employees/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import include
+from django.conf.urls import url
+from rest_framework.routers import DefaultRouter
+from employees import views
```

Comment:
WOW, nice imports! :P

Reply:
No, they are not, they are not alphabetical 😢

Reply:
Done